base/terminfo_data.jl

```diff
@@ -132,7 +133,9 @@ function getcustomalias(allterms::Vector{NTuple{3, String}}, type, short, descri
         "csl"   => ":clear_status_line",
         "Ms"    => ":set_host_clipboard",
         "Tc"    => ":truecolor",
-        "XF"    => ":xterm_focus")
+        "XF"    => ":xterm_focus",
+        "fd"    => ":focus_disable",
+        "fe"    => ":focus_enable")
```

Comment:
Could also do this to make future additions nicer (but obviously that's a matter of taste, too)
```suggestion
        "fe"    => ":focus_enable",
        )
```